include/CLI/App.hpp

```diff
@@ -2408,6 +2573,47 @@ class Option_group : public App {
         add_options(args...);
     }
 };
+/// Helper function to enable one option group/subcommand when another is used
```

Comment:
It's hard to tell here, but these should be in namespace detail, correct? (Or a nameless namespace)

Reply:
I wasn't entirely sure where to put these.  They are intended to used by users similarly to the Validators.  They could be member functions of app.  They could be classes on their own.    I wanted the functionality but I am not sure of the name or location.  Basically a way to turn on/off option groups in a sequence during a parse.

Reply:
Okay, they have a slightly different use than I thought.